l3kernel/l3keys.dtx

```diff
@@ -2518,17 +2517,19 @@
 %    \end{macrocode}
 % \end{macro}
 %
-% \begin{macro}{.initial:n, .initial:V, .initial:o, .initial:x}
+% \begin{macro}{.initial:n, .initial:V, .initial:e, .initial:o, .initial:x}
 %   The standard hand-off approach.
 %    \begin{macrocode}
 \cs_new_protected:cpn { \c_@@_props_root_str .initial:n } #1
   { \@@_initialise:n {#1} }
 \cs_new_protected:cpn { \c_@@_props_root_str .initial:V } #1
   { \exp_args:NV \@@_initialise:n #1 }
+\cs_new_protected:cpn { \c_@@_props_root_str .initial:e } #1
+  { \exp_args:Ne \@@_initialise:n {#1} }
 \cs_new_protected:cpn { \c_@@_props_root_str .initial:o } #1
   { \exp_args:No \@@_initialise:n {#1} }
 \cs_new_protected:cpn { \c_@@_props_root_str .initial:x } #1
-  { \exp_args:Nx \@@_initialise:n {#1} }
+  { \exp_args:Ne \@@_initialise:n {#1} }
```

Comment:
This seems wrong, `.initial:x` should really use `x`-expansion internally.

Reply:
True: must have missed that one cleaning up the search-and-replace changes - reverted.